SIS_import/sis_instructor_practice_course.rb

```diff
@@ -51,7 +51,7 @@ def create_instructor_new_sandbox_site(user_canvas_id, user_name, user_sis_login
 
 	# check again for the current naming format of user sandbox site
 	user_sandbox_site = Canvas_API_GET("#{server_api_url}accounts/#{practice_course_subaccount}/courses?search_term=#{user_sandbox_site_name}")
-	if (user_sandbox_site.length == 0)
+	if (!user_sandbox_site.nil? && user_sandbox_site.length == 0)
```

Comment:
We should either log if user_sandbox_site.nil or create the sandbox site -- do you know which is the correct error response to this?